README.md

```diff
@@ -0,0 +1,43 @@
+# ticket-booking
+
+
+### Additional assumptions
+- Requested time interval for point 1. of the scenario can be at max 1 week.
```

Comment:
Skąd takie założenie?

Reply:
Żeby ograniczyć liczbę jednorazowo pobieranych rekordów. Bez tego użytkownik mógłby pobrać seanse od teraz na kilka miesięcy w przód. Ograniczenie wyszukiwania do jednego tygodnia może być w pewnym sensie paginacją. Dodatkowo uznałem, że użytkownik szukając seansu rzadko potrzebowały dłuższego przedziału.

Reply:
Super, cieszę się, że to zauważyłeś!